### ISSUE TITLE
AlexNet training loss nan

### ISSUE THREAD

**Issue by hoseung2** [pytorch/examples#459]
Hi, i'm training alexnet with imagenet using following command: 

**python main.py -a alexnet /home/imagenet --pretrained**

But after first epoch, it's loss is always nan like below

Epoch: [0][0/5005]      Time 19.235 (19.235)    Data 18.137 (18.137)    Loss 2.0198 (2.0198)    Acc@1 51.562 (51.562)   Acc@5 75.000 (75.000)
Epoch: [0][10/5005]     Time 0.105 (4.681)      Data 0.000 (4.501)      Loss nan (nan)  Acc@1 0.000 (7.209)     Acc@5 0.000 (11.115)
Epoch: [0][20/5005]     Time 17.397 (4.739)     Data 17.346 (4.604)     Loss nan (nan)  Acc@1 0.000 (3.776)     Acc@5 0.000 (5.822)
Epoch: [0][30/5005]     Time 0.105 (4.262)      Data 0.000 (4.141)      Loss nan (nan)  Acc@1 0.000 (2.558)     Acc@5 0.000 (3.944)
Epoch: [0][40/5005]     Time 18.009 (4.412)     Data 17.959 (4.300)     Loss nan (nan)  Acc@1 0.000 (1.934)     Acc@5 0.000 (2.982)
Epoch: [0][50/5005]     Time 0.114 (4.166)      Data 0.000 (4.059)      Loss nan (nan)  Acc@1 0.000 (1.555)     Acc@5 0.000 (2.397)
Epoch: [0][60/5005]     Time 16.965 (4.317)     Data 16.912 (4.214)     Loss nan (nan)  Acc@1 0.000 (1.300)     Acc@5 0.000 (2.004)
Epoch: [0][70/5005]     Time 0.105 (4.153)      Data 0.000 (4.052)      Loss nan (nan)  Acc@1 0.000 (1.117)     Acc@5 0.000 (1.722)
Epoch: [0][80/5005]     Time 15.493 (4.251)     Data 15.445 (4.152)     Loss nan (nan)  Acc@1 0.000 (0.979)     Acc@5 0.000 (1.509)
Epoch: [0][90/5005]     Time 0.105 (4.125)      Data 0.000 (4.027)      Loss nan (nan)  Acc@1 0.000 (0.871)     Acc@5 0.000 (1.344)

python2.7, pytorch 0.4.1

**Comment by soumith** [comment]
see https://github.com/pytorch/examples/tree/master/imagenet#training . we mention that default learning rate has to be reduced for alexnet
